Marlin/src/HAL/HAL_STM32F1/timers.h

```diff
@@ -58,7 +73,7 @@ typedef uint16_t hal_timer_t;
 
 #if MB(BIGTREE_SKR_MINI_E3, BIGTREE_SKR_E3_DIP, BTT_SKR_MINI_E3_V1_2, MKS_ROBIN_LITE)
   // SKR Mini E3 boards use PA8 as FAN_PIN, so TIMER 1 is used for Fan PWM.
-  #define SERVO0_TIMER_NUM 8
+  #define SERVO0_TIMER_NUM 3
```

Comment:
I assume you have actually tested this change with a BLTouch?

Can you confirm that when you do a BLTouch Self-Test from the LCD, the probe deploys exactly 10 times? I found this to be a good indication of whether the timer is working as expected.

Reply:
When I run _Configuration_ -> _BLTouch_ -> _Cmd: Self-Test_, the probe actually deploys and retracts indefinitely. I have 3D Touch from Trianglelabs. I do not know how original BLTouch reacts to this command.

However, I did two rounds of `M48 P50` without a single miss (before your fix, the probe missed one every ~10 probes).
Here are the results:
```
>>> M48 P50
SENDING:M48 P50
M48 Z-Probe Repeatability Test
Finished!
Mean: -0.048850 Min: -0.057 Max: -0.043 Range: 0.015
Standard Deviation: 0.002849
>>> M48 P50
SENDING:M48 P50
M48 Z-Probe Repeatability Test
Finished!
Mean: 0.263475 Min: 0.257 Max: 0.270 Range: 0.013
Standard Deviation: 0.003599
```

Reply:
OK, the genuine BLTouch deploys and retracts 10 times during self-test.
Prior to my first fix for this, it would stop early because the Servo signal changed when it was not supposed to.

Your M48 testing sounds good to me.

Reply:
Can (should) we continue to use timer 8 in the case of `#ifndef STM32_HIGH_DENSITY` ?